scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -37,3 +37,15 @@ Feature: Deploying a Daemon environment to a cluster with Replace After Terminat
       | containerInstanceArn | taskDefinitionArn | group             |
       | i-1                  | v1                | DaemonEnvironment |
     And it should not take any further actions
+
+  Scenario: Scheduling on a cluster with tasks that don't match the environment version
+    Given the cluster has the following instances and tasks:
+      | instance | tasks          |
+      | i-1      | t-1:v1:RUNNING |
+      | i-2      | t-2:v2:RUNNING |
+      | i-3      |                |
+    When the scheduler runs
+    Then it should stop the following tasks:
+      | clusterName | task           | reason                |
+      | TestCluster | v2             | replaceAfterTerminate |
+    And it should not take any further actions
```

Comment:
Shouldn't this also start a task for `i-3`?

Reply:
As discussed, the shceduler will stop the task on i-2 and start a task on i-3. And I will make it a separate case for testing this multiple step scheduling results.